### ISSUE TITLE
Column in document switcher reset size/position when resized

### ISSUE THREAD

**Issue by shravan2x** [notepad-plus-plus/notepad-plus-plus#4715]
### Description of the Issue

Column in document switcher reset size/position when resized:

![npp-issue](https://user-images.githubusercontent.com/4096676/43438952-544eedf4-9456-11e8-85cb-a14d02e3a80d.gif)

### Debug Information
```
Notepad++ v7.5.8   (64-bit)
Build time : Jul 23 2018 - 02:06:36
Path : C:\Program Files\Notepad++\notepad++.exe
Admin mode : OFF
Local Conf mode : OFF
OS : Windows 10 (64-bit)
Plugins : ComparePlugin.dll NPPJSONViewer-x64.dll PluginManager.dll DSpellCheck.dll mimeTools.dll NppConverter.dll 
```

**Comment by shimscharf** [comment]
This is one of the most annoying bugs. 
I frequently have more than 20 documents open, and I can't see the extension of the files.
After correcting column width, it just defaults back to the original width every time I tab back into Notepad++ OR when Notepad++ is minimized then reopened

Notepad++ v7.7.1   (32-bit)
Build time : Jun 16 2019 - 21:14:50
Path : C:\Program Files (x86)\Notepad++\notepad++.exe
Admin mode : OFF
Local Conf mode : OFF
OS : Windows 10 (64-bit)
Plugins : ComparePlugin.dll JSMinNPP.dll mimeTools.dll NppConverter.dll NppExport.dll SessionMgr.dll

**Comment by ArkadiuszMichalski** [comment]
I can confirm this in:
```
Notepad++ v7.9   (32-bit)
Build time : Sep 14 2020 - 22:58:18
Path : D:\_test\NPPClean\notepad++.exe
Admin mode : ON
Local Conf mode : ON
OS Name : Windows 7 Ultimate (64-bit) 
OS Build : 7601.0
Current ANSI codepage : 1250
Plugins : none
```

**Comment by donho** [comment]
@ArkadiuszMichalski 
I cannot reproduce this issue in v8.1.4, can you?

**Comment by shimscharf** [comment]
I can confirm it still occurs in version 8.1.2

Notepad++ v8.1.2   (64-bit)
Build time : Jul 16 2021 - 01:59:50
Path : C:\Program Files\Notepad++\notepad++.exe
Command Line : 
Admin mode : OFF
Local Conf mode : OFF
Cloud Config : OFF
OS Name : Windows 10 Pro (64-bit) 
OS Version : 2009
OS Build : 19043.1165
Current ANSI codepage : 1252
Plugins : ComparePlugin.dll JSMinNPP.dll mimeTools.dll NppConverter.dll NppExport.dll NppTaskList.dll SessionMgr.dll

**Comment by donho** [comment]
@shimscharf 
I'm talking about **v8.1.4**
Not v8.1.2 !

**Comment by ArkadiuszMichalski** [comment]
@donho 
I guess what I meant was: when we change the width of the `Ext.` and change the width of the `Document List`, then `Ext.` returns to the original width and is additionally placed furthest to the right. But the lack of visibility of `Ext.` I can't reproduce. Maybe something with resolution / scaling dependent again.

@@shimscharf
Try portable https://github.com/notepad-plus-plus/notepad-plus-plus/wiki/Testing#using-the-latest-release-portable-version-of-notepad.

```
Notepad++ v8.1.4   (32-bit)
Build time : Aug 21 2021 - 12:58:42
Path : C:\Users\Arek\Desktop\NPPClean\notepad++.exe
Command Line : -multiInst
Admin mode : ON
Local Conf mode : ON
Cloud Config : OFF
OS Name : Windows 7 Ultimate (64-bit) 
OS Build : 7601.0
Current ANSI codepage : 1250
Plugins : HexEditor.dll mimeTools.dll NppConverter.dll NppExport.dll 
```

**Comment by shimscharf** [comment]
It is FIXED in 8.1.3

Notepad++ v8.1.3   (64-bit)
Build time : Aug 10 2021 - 00:29:32
Path : C:\Program Files\Notepad++\notepad++.exe
Command Line : 
Admin mode : OFF
Local Conf mode : OFF
Cloud Config : OFF
OS Name : Windows 10 Pro (64-bit) 
OS Version : 2009
OS Build : 19043.1165
Current ANSI codepage : 1252
Plugins : ComparePlugin.dll JSMinNPP.dll mimeTools.dll NppConverter.dll NppExport.dll NppTaskList.dll SessionMgr.dll

**Comment by donho** [comment]
It seems also the high dpi issue to me too. This issue has been fixed in v8.1.4. So normally, Ext column will display correctly in v8.1.4 under high dpi display. However, resized column will be restored back after resizing the panel. This issue won't be enhanced.

**Comment by shimscharf** [comment]
This also used to occur when minimizing notepad++, and then restoring it. It doesn't occur any more - it is fixed as of 8.1.3
